doc/devel/index.rst

```diff
@@ -31,9 +44,7 @@ and fixing bugs and requesting and implementing new features...
       :octicon:`bug;1em;sd-text-info` **Submit a bug report**
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-      We have preloaded the issue creation page with a Markdown form that you can
-      use to provide relevant context. Thank you for your help in keeping bug reports
-      complete, targeted and descriptive.
+      Thank you for your help in keeping bug reports complete, targeted and descriptive.
```

Comment:
```suggestion
      Thank you for your help in keeping bug reports targeted and descriptive.
```
Main motivation is to bring both boxes to the same number of lines so that the buttons align. I believe "completed" is mostly handled by descriptive. I'm not putting in too much thought on the exact wording, but I believe the suggestion is ok. If you want something else, go for it.